### ISSUE TITLE
Error when generating graph

### ISSUE THREAD

**Issue by davidjirovec** [mdaines/viz.js#54]
When I try to generate graph for
`digraph G { rankdir="LR" subgraph cluster_ORIGINATOR { DRAFT; } subgraph cluster_MRA_REVIEWER { PENDING_REJECT; PENDING_ASSIGN; DISCARDED; } subgraph cluster_MRA_PRIMARY_APPROVER { ASSIGNED; } subgraph cluster_MRA_SECONDARY_APPROVER { PENDING_DEPRECATE; APPROVED; DISALLOWED; APPROVED_AUTOMATICALLY; RETIRED; REJECTED; PENDING_RETIRE; PENDING_FINAL_REJECT; PENDING_APPROVE; PENDING_NOT_REQUIRED; DEPRECATED; PENDING_DISALLOW; } DRAFT -> PENDING_ASSIGN [label="SUBMIT"];DRAFT -> ASSIGNED [label="REWORKED_DRAFT_SUBMIT"];ASSIGNED -> PENDING_FINAL_REJECT [label="FINAL_REJECT"];ASSIGNED -> PENDING_DEPRECATE [label="DEPRECATE"];ASSIGNED -> PENDING_NOT_REQUIRED [label="NOT_REQUIRED"];ASSIGNED -> DRAFT [label="REVERT"];ASSIGNED -> DISCARDED [label="TERMINATE"];ASSIGNED -> PENDING_RETIRE [label="RETIRE"];ASSIGNED -> PENDING_DISALLOW [label="DISALLOW"];ASSIGNED -> PENDING_REJECT [label="REJECT"];ASSIGNED -> PENDING_APPROVE [label="APPROVE"];ASSIGNED -> ASSIGNED [label="SAVE_AS_DRAFT"];PENDING_REJECT -> ASSIGNED [label="REVIEWER_SUBMIT"];PENDING_REJECT -> DRAFT [label="REVERT"];PENDING_REJECT -> DISCARDED [label="AGREE_DISCARD"];PENDING_ASSIGN -> DRAFT [label="REVERT"];PENDING_ASSIGN -> ASSIGNED [label="REVIEWER_SUBMIT"];PENDING_ASSIGN -> DISCARDED [label="TERMINATE"];PENDING_NOT_REQUIRED -> APPROVED_AUTOMATICALLY [label="AGREE_NOT_REQUIRED"];PENDING_NOT_REQUIRED -> DRAFT [label="REVERT"];PENDING_NOT_REQUIRED -> DISCARDED [label="TERMINATE"];PENDING_NOT_REQUIRED -> ASSIGNED [label="RESET"];PENDING_APPROVE -> DRAFT [label="REVERT"];PENDING_APPROVE -> ASSIGNED [label="RESET"];PENDING_APPROVE -> DISCARDED [label="TERMINATE"];PENDING_APPROVE -> APPROVED [label="AGREE_APPROVE"];PENDING_FINAL_REJECT -> REJECTED [label="AGREE_REJECT"];PENDING_FINAL_REJECT -> DRAFT [label="REVERT"];PENDING_FINAL_REJECT -> DISCARDED [label="TERMINATE"];PENDING_FINAL_REJECT -> ASSIGNED [label="RESET"];PENDING_DEPRECATE -> DRAFT [label="REVERT"];PENDING_DEPRECATE -> DISCARDED [label="TERMINATE"];PENDING_DEPRECATE -> ASSIGNED [label="RESET"];PENDING_DEPRECATE -> DEPRECATED [label="AGREE_DEPRECATE"];PENDING_DISALLOW -> DRAFT [label="REVERT"];PENDING_DISALLOW -> DISCARDED [label="TERMINATE"];PENDING_DISALLOW -> DISALLOWED [label="AGREE_DISALLOW"];PENDING_RETIRE -> ASSIGNED [label="RESET"];PENDING_RETIRE -> DRAFT [label="REVERT"];PENDING_RETIRE -> DISCARDED [label="TERMINATE"];PENDING_RETIRE -> RETIRED [label="AGREE_RETIRE"];}`

I get an error:
`Uncaught abort() at Error at jsStackTrace (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:20800) at stackTrace (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:20983) at Object.abort (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:28:5424) at _abort (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:603400) at O4 (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:21:14501) at zf (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:11:124669) at Array.sf (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:11:120869) at gF (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:9:172965) at ec (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:11:882) at Object.ccallFunc [as ccall] (http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:8080) If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.`

Can be tested on http://mdaines.github.io/viz.js/

When I generate graph for same input using Graphviz dot.exe binary, it works ok.

**Comment by LBierkandt** [comment]
I encountered the same error; perhaps another example helps pinning down the problem:

I got the error when trying to generate the following graph:
```dot
digraph G{
	type="digraph";
	rankdir="TB";
	3->5[label="e1",color="#00cc00",constraint="false",fontcolor="#00cc00"]
	5->4[label="e2",color="#00cc00",constraint="false",fontcolor="#00cc00"]
	10->5[label="e3",color="#00cc00",constraint="false",fontcolor="#00cc00"]
	10->8[label="e6",color="#00cc00",constraint="false",fontcolor="#00cc00"]
	3->4[style="invis"]
	4->5[style="invis"]
	5->6[style="invis"]
	6->7[style="invis"]
	7->8[style="invis"]
	8->9[style="invis"]
	9->10[style="invis"]
	10->11[style="invis"]
	subgraph tokens{
		rank="same";
		3[label="t1",shape="box"]
		4[label="t2",shape="box"]
		5[label="t3",shape="box"]
		6[label="t4",shape="box"]
		7[label="t5",shape="box"]
		8[label="t6",shape="box"]
		9[label="t7",shape="box"]
		10[label="t8",shape="box"]
		11[label="t9",shape="box"]
	}
}
```
The stacktrace:
`
InternalError: uncaught exception: abort() at jsStackTrace@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:20800 stackTrace@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:20983 abort@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:28:5424 _abort@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:603385 O4@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:21:14501 zf@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:11:124669 sf@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:11:120869 gF@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:9:172924 ec@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:11:882 ccallFunc@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:5:8075 render@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:62:25 Viz@http://mdaines.github.io/viz.js/bower_components/viz.js/viz.js:44:14 onmessage@http://mdaines.github.io/viz.js/worker.js:4:16 If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.
`

Graph generation works as soon as you remove one of the visible edges.

**Comment by LBierkandt** [comment]
This is actually an instance of the bug reported in #65.

**Comment by mdaines** [comment]
Addressed in 2257a4017ed3bdc4bbf635279d9cd066091a425b.
